app/controllers/api/base_controller/authentication.rb

```diff
@@ -64,14 +63,14 @@ def api_token_mgr
       end
 
       def authenticate_with_user_token(x_auth_token)
-        @auth_token = x_auth_token
-        if !api_token_mgr.token_valid?(@auth_token)
-          raise AuthenticationError, "Invalid Authentication Token #{@auth_token} specified"
+        auth_token = x_auth_token
```

Comment:
Good catch! I tackled in https://github.com/ManageIQ/manageiq/pull/15016/commits/d1b438f94500763ba0f73c93dbdf049f37e073b0